apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -1219,7 +1222,22 @@ handle_cm_partition_table_response({ok, {{<<"200">>, _}, _, Body, _, _}}) ->
 								DecodedPartition = {
 									Bucket,
 									BucketSize,
-									ar_util:decode(EncodedAddr)
+									ar_util:decode(EncodedAddr),
+									0
+								},
+								{ok, [DecodedPartition | Acc]};
+							{[
+								{<<"bucket">>, Bucket},
+								{<<"bucketsize">>, BucketSize},
+								{<<"addr">>, EncodedAddr},
+								{<<"pdiff">>, PackingDifficulty}
+							]} when is_integer(PackingDifficulty), PackingDifficulty >= 0,
```

Comment:
Should this check >= 1? It looks like we only serialize to `{pdiff, xxx}` if the difficlty is >= 1, so if we get a 0 here it might indicate some weird error?